data/processed/ref/total_4_fte.yaml

```diff
@@ -0,0 +1,23 @@
+api: fetch
+api_documentation: https://www.ref.ac.uk/2014/pubs/201401/
+description: These are the results of the Research Excellence Framework, where university departments in various disciplines are assessed on the quality of their research. The latest REF was conducted in 2014 so data is available only for one year
+endpoint_static: https://results.ref.ac.uk/(S(hlvnuqzwkag44jp3df3d4q14))/DownloadFile/AllResults/xlsx
+framework_group: public_rnd_capability
+is_experimental: False
+nuts_level: 2
+source: the data provider: https://results.ref.ac.uk/
+data_date: 20200204
+year_range: [2014, 2014]
+schema:
+  nuts_id:
+    type: NutsRegion.id
+  nuts_year_spec:
+    type: NutsRegion.year_spec
+  value:
+    description: The total number of excellent researchers (4* score) submitted by universities in the NUTS-2 region for the 2014 REF
+    id: mean_ref
+    label: Total number of excellent researchers in REF
+    type: total_4_fte
```

Comment:
I think this is now `FTE`. Also please add a `format`, here or to the `FTE` type itself if it maks sense.